cypress/component/utils/utils.spec.js

```diff
@@ -17,6 +18,53 @@ const collectionsSkeleton = [
   }
 ];
 
+const collectionWithMixedElectionStatuses = {
+  dars: {
+    0: {
```

Comment:
Why key this by index? Would an array be a better object for `dars`?

Reply:
Real dar collection data is a map of dar reference id to dar. Similarly, elections are a map of election id to election, etc.